ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyOptimizer.cs

```diff
@@ -397,23 +397,6 @@ private void ApplyLayoutBase(ReactShadowNode node)
                 parent = parent.Parent;
             }
 
-            // This is a hack that accomodates for the fact that borders are
-            // wrapped around the canvases that contain the UI elements. It is
-            // likely to prove brittle over time, and we should consider either
-            // alternate ways of drawing borders, or different mechanisms to
-            // set absolute positions of elements.
-            var borderParent = node.Parent;
-            if (borderParent != null && borderParent.IsLayoutOnly)
-            {
-                borderParent = node.NativeParent;
-            }
-
-            if (borderParent != null)
-            {
-                x -= borderParent.GetLeftBorderWidth();
```

Comment:
>GetLeftBorderWidth [](start = 34, length = 18)

Can you also delete these helpers, which are now no longer needed? I.e., delete ReactShadowNodeExtensions.cs.